### DISCUSSION TITLE
Solve login issue

### DISCUSSION THREAD

**Discussion by ayanarshad** [SideStore/SideStore#765] [Show and tell]
Login with apple error occurs please solve this

**Comment by ghost** [comment]
change anisette server

**Comment by elchepas31-png** [comment]
I need help!!

when I try to log in it says I need a 6 digit code but I don’t have any, I tried many times 
<img width="828" height="1792" alt="IMG_3683" src="https://github.com/user-attachments/assets/32649bd1-f34b-4fff-bf17-c788163c7760" />

**Reply by neoarz** [reply]
> I need help!!
> 
> when I try to log in it says I need a 6 digit code but I don’t have any, I tried many times <img alt="IMG_3683" width="828" height="1792" src="https://private-user-images.githubusercontent.com/226370045/477679935-32649bd1-f34b-4fff-bf17-c788163c7760.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTUxMDg1MzgsIm5iZiI6MTc1NTEwODIzOCwicGF0aCI6Ii8yMjYzNzAwNDUvNDc3Njc5OTM1LTMyNjQ5YmQxLWYzNGItNGZmZi1iZjE3LWM3ODgxNjNjNzc2MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgxM1QxODAzNThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YmM0ODE2ZTliZmMxZjAzNWQ5YTFmNmQ4NGI2OTAxYjExNzdmM2QyOGFiNTI0NWRkNTQwNzIxY2FjODdjYjI5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.cPudt6lDi0HVKdmCYqt8FMYKLj1tRa90VUjzwdb5He4">

https://docs.sidestore.io/docs/troubleshooting/#verification-code-not-received-when-signing-in-with-apple-id

**Comment by ghost** [comment]
https://discord.gg/sidestore-949183273383395328
